src/index.es6

```diff
@@ -69,6 +70,16 @@ function SamJs (options) {
   this.download = (text, phonetic) => {
     RenderBuffer(this.buf8(text, phonetic));
   }
+
+  /**
+   * Render the passed text as wave buffer array.
+   * 
+   * @param {string}  text       The text to render or phoneme string.
+   * @param {boolean} [phonetic] Flag if the input text is already phonetic data.
+   * 
+   * @return {Uint8Array}
+   */
+  this.wav = (text, phonetic) => ToWavBuffer(this.buf8(text, phonetic));
```

Comment:
The types aren't very precise since `ToWavBuffer` expects an `Uint8Array` but `this.buf8` can return a `boolean`

Reply:
I guess we should denote this then as:
```
@return {Uint8Array|bool}
```
Or maybe even
```
@return {Uint8Array|false}
```
I do not know about jsdoc support for the latter one.